apps/vue-showcase/cypress.config.ts

```diff
@@ -15,5 +15,12 @@ export default defineConfig({
   e2e: {
     ...nxE2EPreset(__dirname),
     ...cypressJsonConfig,
+    /**
+     * TODO(@nrwl/cypress): In Cypress v12,the testIsolation option is turned on by default.
+     * This can cause tests to start breaking where not indended.
+     * You should consider enabling this once you verify tests do not depend on each other
+     * More Info: https://docs.cypress.io/guides/references/migration-guide#Test-Isolation
+     **/
+    testIsolation: false,
```

Comment:
Have you planned on testing this in the future? Having a ticket on the board is good enough.

Reply:
Ah, this came with an NX migration. Nice noticing it. I'll just remove these and run the tests, they should be isolated already, but this would reveal if they wouldn't be.

Reply:
The vue-showcase tests fail if I remove this. I'll create a ticket for it.

Reply:
I created #161 to fix the tests with testIsolation true.